pymc3/distributions/continuous.py

```diff
@@ -4226,6 +4226,31 @@ class Interpolated(BoundedContinuous):
     ========  ===========================================
     Support   :math:`x \in [x\_points[0], x\_points[-1]]`
     ========  ===========================================
+    
```

Comment:
The example should go above the support thinghy and inside a `.. plot::` directive. You can see one example of this at: https://github.com/pymc-devs/pymc3/blob/master/pymc3/distributions/continuous.py#L193